index.html

```diff
@@ -15,9 +15,7 @@
 <body>
     <!--HEADER DESIGN-->
     <header class="header">
-        <a href="" class="logo">Grupo <span>Pires</span></a>
```

Comment:
evite deixar href vazio, pois ao clicar, a tela fica piscando. Se não for levar a algum lugar, use apenas uma #